resources/definitions/cartesio.def.json

```diff
@@ -0,0 +1,52 @@
+{
+    "id": "cartesio",
+    "name": "Cartesio",
+    "version": 2,
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Scheepers",
+        "manufacturer": "Cartesio bv",
+        "category": "Other",
+        "file_formats": "text/x-gcode",
+        "has_materials": true,
+        "has_machine_materials": true,
```

Comment:
We are not really comfortable with the current standard materials, and we do not want customers to pick them. We are making material files with machine/nozzle specific settings that they can download